exchanges/stream/websocket_test.go

```diff
@@ -18,6 +18,7 @@ import (
 
```

Comment:
Once `TestMain` is executing tests again, I got the following race
```
==================
WARNING: DATA RACE
Write at 0x000104f1c970 by goroutine 10:
  github.com/thrasher-corp/gocryptotrader/exchanges/stream.TestTrafficMonitorTimeout()
      /Users/a/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/stream/websocket_test.go:252 +0xb28
  testing.tRunner()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:1689 +0x180
  testing.(*T).Run.gowrap1()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:1742 +0x40

Previous read at 0x000104f1c970 by goroutine 66:
  github.com/thrasher-corp/gocryptotrader/exchanges/stream.(*Websocket).trafficMonitor.func1()
      /Users/a/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/stream/websocket.go:569 +0x74

Goroutine 10 (running) created at:
  testing.(*T).Run()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:1742 +0x5e4
  testing.runTests.func1()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:2161 +0x80
  testing.tRunner()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:1689 +0x180
  testing.runTests()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:2159 +0x6e0
  testing.(*M).Run()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:2027 +0xb6c
  github.com/thrasher-corp/gocryptotrader/exchanges/stream.TestMain()
      /Users/a/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/stream/websocket_test.go:112 +0x44
  main.main()
      _testmain.go:111 +0x294

Goroutine 66 (running) created at:
  github.com/thrasher-corp/gocryptotrader/exchanges/stream.(*Websocket).trafficMonitor()
      /Users/a/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/stream/websocket.go:557 +0xd0
  github.com/thrasher-corp/gocryptotrader/exchanges/stream.(*Websocket).Connect()
      /Users/a/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/stream/websocket.go:285 +0x314
  github.com/thrasher-corp/gocryptotrader/exchanges/stream.TestConnectionMessageErrors()
      /Users/a/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/stream/websocket_test.go:315 +0x740
  testing.tRunner()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:1689 +0x180
  testing.(*T).Run.gowrap1()
      /opt/homebrew/Cellar/go/1.22.0/libexec/src/testing/testing.go:1742 +0x40
==================
--- FAIL: TestTrafficMonitorTimeout (0.58s)
    testing.go:1398: race detected during execution of test
--- FAIL: TestGenerateMessageID (0.63s)
    testing.go:1398: race detected during execution of test
--- FAIL: TestDial (0.00s)
    --- FAIL: TestDial/test1 (1.14s)
        testing.go:1398: race detected during execution of test
--- FAIL: TestSendMessage (0.00s)
    --- FAIL: TestSendMessage/test1 (1.17s)
        testing.go:1398: race detected during execution of test
--- FAIL: TestWebsocketConnectionShutdown (1.18s)
    testing.go:1398: race detected during execution of test
--- FAIL: TestSendMessageWithResponse (1.18s)
    testing.go:1398: race detected during execution of test
--- FAIL: TestLatency (2.18s)
    testing.go:1398: race detected during execution of test
--- FAIL: TestSetupPingHandler (2.56s)
    testing.go:1398: race detected during execution of test
FAIL
exit status 1
```

Reply:
Already fixed 7dd00dc9 😄 
It was due to the other setting of `trafficCheckInterval`.
Sloppy last minute fix from me. Sorry.